public/js/template.js

```diff
@@ -0,0 +1,38 @@
+jQuery(document).ready(function($) {
+
+	var my_nav = $('.navbar-sticky'); 
+	// grab the initial top offset of the navigation 
+	var sticky_navigation_offset_top = my_nav.offset().top;
+	
+	// our function that decides weather the navigation bar should have "fixed" css position or not.
+	var sticky_navigation = function(){
+		var scroll_top = $(window).scrollTop(); // our current vertical position from the top

```

Comment:
Codacy found an issue: [Identifier 'scroll_top' is not in camel case.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6084628)